Vagrantfile

```diff
@@ -11,16 +11,23 @@ Vagrant.configure(2) do |config|
     config.vbguest.auto_update = true
   end
 
-  config.vm.define "dcos-centos-virtualbox" do |vm_cfg|
-    vm_cfg.vm.box = $dcos_box
-    vm_cfg.vm.box_url = $dcos_box_url
-    vm_cfg.vm.box_version = $dcos_box_version
+  config.vm.hostname = 'dcos-centos'
+  config.vm.box = $dcos_box
+  config.vm.box_url = $dcos_box_url
+  config.vm.box_version = $dcos_box_version
 
-    vm_cfg.vm.provider "virtualbox" do |v|
-      v.name = "dcos-centos-virtualbox"
-      v.cpus = 2
-      v.memory = 2048
-      v.customize ['modifyvm', :id, '--natdnshostresolver1', 'on']
-    end
+  config.vm.provider "virtualbox" do |vb|
+    vb.name = "dcos-centos-virtualbox"
+    vb.cpus = 2
+    vb.memory = 2048
+    vb.customize ['modifyvm', :id, '--natdnshostresolver1', 'on']
+    vb.linked_clone = true
```

Comment:
@yankcrime Learning for me! Should I use this in minidcos?

Reply:
I see from the docs that when this is used:

> Note: the generated master VMs are currently not removed automatically by Vagrant. This has to be done manually. However, a master VM can only be removed when there are no linked clones connected to it.

Is this accounted for somehow? Or will users now have to know somehow to clean up?

Reply:
I'd recommend enabling it, yes - it does noticeably reduce startup time for new machines.  Users will have to remove the old master Vagrant box as / when they upgrade after we publish a new version.  I'll add something to the README along those lines.